src/Config.php

```diff
@@ -1,51 +1,52 @@
 <?php
+
 namespace Pantheon\Terminus;
 
 use Dotenv\Dotenv;
 use Symfony\Component\Yaml\Yaml;
 
 class Config
 {
-  /**
-   * @var string[]
-   */
+    /**
```

Comment:
Funny question: is 4 spaces the proper amount to indent for classes when the `{` is on the next line?